addons/account_audit_trail/models/mail_message.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from markupsafe import Markup
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class Message(models.Model):
+    _inherit = 'mail.message'
+
+    account_audit_log_preview = fields.Html(string="Description", compute="_compute_account_audit_log_preview")
+    account_audit_log_move_id = fields.Many2one(
+        comodel_name='account.move',
+        string="Journal Entry",
+        compute="_compute_account_audit_log_move_id",
+        search="_search_account_audit_log_move_id",
+    )
+    show_audit_log = fields.Boolean(compute="_compute_account_audit_log_move_id", search="_search_show_audit_log")
+
+    @api.depends('body', 'subject', 'tracking_value_ids', 'subtype_id')
+    def _compute_account_audit_log_preview(self):
+        for message in self:
+            message_sudo = self.browse(message['id']).sudo().with_prefetch(self.ids)
+            title = message.subject or message.preview
+            tracking_value_ids = message_sudo.tracking_value_ids.filtered(lambda tracking: not tracking.field_groups or self.env.is_superuser() or self.user_has_groups(tracking.field_groups))
+            if not title and tracking_value_ids:
+                title = _("Updated")
+            elif not title and message.subtype_id and not message.subtype_id.internal:
+                title = message.subtype_id.display_name
```

Comment:
No need to `browse` and `with_prefetch` when you already have a recordset.
It only makes sense in `mail` because it comes from a dict of raw values
I also find it easier to read to have the if/elif/else written like that
```suggestion
            tracking_value_ids = message.sudo().tracking_value_ids.filtered(lambda tracking: not tracking.field_groups or self.env.is_superuser() or self.user_has_groups(tracking.field_groups))
            if not title and tracking_value_ids:
                title = _("Updated")
            elif not title and message.subtype_id and not message.subtype_id.internal:
                title = message.subtype_id.display_name
            else:
                title = message.subject or message.preview
```

Reply:
@william-andre Can't move `title = message.subject or message.preview`, because it's reference is needed for if/elif statement.

Reply:
Ok, just change the sudo and prefetch thing then